test/unit/helpers.lua

```diff
@@ -68,14 +68,12 @@ local function cimport(...)
 
   local body = nil
   for _ = 1, 10 do
-    local stream = Preprocess.preprocess_stream(unpack(paths))
-    body = stream:read("*a")
-    stream:close()
+    body = Preprocess.preprocess(unpack(paths))
     if body ~= nil then break end
   end
 
   if body == nil then
-    print("ERROR: helpers.lua: Preprocess.preprocess_stream():read() returned empty")
+    print("ERROR: helpers.lua: Preprocess.preprocess() returned empty")
```

Comment:
This sanity check was for the `:read()` call. Because of a lua or lfs bug it sometimes returns empty, and catching it here saves debugging later. Not sure if it's still needed, but if so it so check `ret == nil` after line 182 in `test/unit/preprocess.lua`.